dcos-log/dcos-log.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/dcos/dcos-go/dcos-log/api"
+	"github.com/dcos/dcos-go/dcos-log/config"
+)
+
+func main() {
+	cfg, err := config.NewConfig(os.Args)
+	if err != nil {
+		logrus.Fatalf("Could not load config: %s", err)
+	}
+
+	if cfg.FlagVerbose {
```

Comment:
Can we move this into the config file instead of main()? That way we could *possibly test that the flag setting toggles this (if we wanted to test that).